apps/wallet-mobile/src/features/Discover/useCases/BrowseDapp/BrowserToolbar.tsx

```diff
@@ -16,11 +16,11 @@ export const BrowserToolbar = ({uri}: Props) => {
   const {isSecure, domainName} = getDomainFromUrl(uri)
 
   const handleCloseBrowser = () => {
-    navigateTo.discover()
+    navigateTo.selectDappFromList()
   }
 
   const handleEditUrl = () => {
```

Comment:
handle edit and navigates to search, is that correct? Guessing: if the user taps on location we navigate to a search?